src/helpers/http.ts

```diff
@@ -12,22 +12,28 @@ export class HttpClient {
 
   get = async <T>(url: string, config = {}) => {
     const response = await this.client.get<T>(url, config).catch((err) => {
-      throw new Error(err)
+      if (err.response) throw { status: err.response.status, data: err.response.data }
+      throw { status: 500, data: "ECONNREFUSED", error: err }
+      // throw new Error('url:' + url + ' Error:' + err)
```

Comment:
this new answer structure should be considered and marked as a breaking change from a developer's pov.

Same for the two remaining ones.